wa/commands/process.py

```diff
@@ -0,0 +1,84 @@
+#    Copyright 2014-2018 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+import sys
+
+from wa import Command, settings
+from wa import discover_wa_outputs
+from wa.framework import pluginloader
+from wa.framework.version import get_wa_version
+from wa.framework.output_processor import OutputProcessor
+from wa.utils.log import log_error
+
+class ProcessCommand(Command):
+
+    name = 'process'
+    description = 'Process the output from previously run workloads.'
+
+    def __init__(self, subparsers):
+        self.loader = pluginloader
+        super(ProcessCommand, self).__init__(subparsers)
+
+    def initialize(self, context):
+        self.parser.add_argument('-d', '--process-directory', metavar='DIR', default=None,
+                                 help="""
+                                 Specify a directory to walk to find the data
+                                 from previous runs to process .If this option is
+                                 not specified, then {} will be used instead.
```

Comment:
Add double grave quotes (` `` `) around  the path to make sure it displays as monospace.